Sources/OpenTelemetryApi/Metrics/Stable/DefaultStableMeter.swift

```diff
@@ -123,7 +131,15 @@ public class DefaultStableMeter : StableMeter {
     func add(value: Int, attribute: [String : AttributeValue]) {}
   }
   
-  private class NoopLongCounterBuilder : LongCounterBuilder {
+    private class NoopLongCounterBuilder : LongCounterBuilder {
```

Comment:
setDescription & setUnit are accessible via the InstrumentBuilder protocol, remove these.